pkg/controller/directvolumemigration/rsync.go

```diff
@@ -159,16 +159,77 @@ func (t *Task) getRsyncClientMutations(client compat.Client, namespace string) (
 	if err != nil {
 		return nil, liberr.Wrap(err)
 	}
-	runAsUser := int64(0)
-	trueBool := bool(true)
-	customSecurityContext := &corev1.SecurityContext{
-		Privileged:             &isPrivileged,
-		RunAsUser:              &runAsUser,
-		ReadOnlyRootFilesystem: &trueBool,
-		Capabilities: &corev1.Capabilities{
-			Drop: []corev1.Capability{"MKNOD", "SETPCAP"},
-		},
+
+	destClient, err := t.getDestinationClient()
```

Comment:
why not pass both clients to the function, they are already available in the caller?